java/kotlin-lite/BUILD

```diff
@@ -1,7 +1,7 @@
 load("@io_bazel_rules_kotlin//kotlin:jvm.bzl", "kt_jvm_library")
 load("@rules_java//java:defs.bzl", "java_lite_proto_library")
 load("@rules_jvm_external//:kt_defs.bzl", "kt_jvm_export")
-load("//:protobuf_version.bzl", "PROTOBUF_VERSION")
+load("//:protobuf_version.bzl", "PROTOBUF_KOTLIN_VERSION")
```

Comment:
I don't see us decoupling PROTOBUF_JAVA_VERSION and PROTOBUF_KOTLIN_VERSION since kotlin is dependent on java. I think we should just use a single PROTOBUF_JAVA_VERSION